documentation/src/main/asciidoc/mapping.asciidoc

```diff
@@ -1475,22 +1473,36 @@ public class Meeting {
     // ...
 ----
 ====
-
-You can also choose to encode the date as string using the `encoding=EncodingType.STRING` of
-`DateBridge`. In this case the dates are stored in the format _yyyyMMddHHmmssSSS_ (using GMT time).
-
++
 [IMPORTANT]
 ====
 A Date whose resolution is lower than `MILLISECOND` cannot be a `@DocumentId`
 ====
-
++
+You can also choose to encode the date as string using the `encoding=EncodingType.STRING` of
+`DateBridge`. In this case the dates are stored in the format _yyyyMMddHHmmssSSS_ (using GMT time).
```

Comment:
`**@**DateBridge` to be consistent with above. And maybe mention `@CalendarBridge` too.